### PR TITLE
Update the npy package

### PR DESCRIPTION
**Pull request by LaurentMazare**
This adds support for dune 2.0 to the npy package.

### CODE REVIEW

File: packages/npy/npy.0.0.9/opam [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+bug-reports:  "https://github.com/LaurentMazare/npy-ocaml/issues"
+homepage:     "https://github.com/LaurentMazare/npy-ocaml"
+dev-repo:     "git+https://github.com/LaurentMazare/npy-ocaml.git"
+maintainer:   "Laurent Mazare <lmazare@gmail.com>"
+authors:      [ "Laurent Mazare" ]
+
+build: [["dune" "build" "-p" name "-j" jobs]]
+depends: [
+  "camlzip"
+  "dune"
```
Comment (kit-ty-kate) [inline packages/npy/npy.0.0.9/opam]:
```suggestion
  "dune" {>= "1.4"}
```
According to your `dune-project` file, this package requires at least dune 1.4

**Comment by LaurentMazare** [comment]
@kit-ty-kate  I just added your dune>=1.4 suggestion, thanks for pointing this out.

**Comment by LaurentMazare** [comment]
Anything more I should do on this @kit-ty-kate ?

**Comment by kit-ty-kate** [comment]
No, sorry for the wait, I forgot about this PR. Thanks!

**Comment by LaurentMazare** [comment]
Thanks!

**Pull request merged**